MuteSpotifyAds/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -1,63 +1,63 @@
 {
-  "images": [
+  "images" : [
     {
-      "size": "16x16",
-      "idiom": "mac",
-      "filename": "icon_16.png",
-      "scale": "1x"
+      "size" : "16x16",
```

Comment:
Please do not change the formatting of JSON files.

Reply:
i believe xcode automatically did this when i changed my icons to match the ones on github, i'll do a commit to revert those